java-api-core/src/com/kinvey/java/network/NetworkFileManager.java

```diff
@@ -729,7 +736,7 @@ private DownloadMetadataAndFile(FileMetaData meta, DownloaderProgressListener pr
 
         @Key("id")
         private String id;
-        
+
         @Key("query")
         private String queryFilter;
         @Key("sort")
```

Comment:
I think for `inner` classes in `Java` better use `inner static` classes, e.g.:
 ```public static class UploadMetadataAndFile...```
To avoid possibility of memory leaks.

Or maybe better convert them to `Kotlin` later where cannot be such problem if we declare classes that way.

Reply:
just converted the entire class to Kotlin, since it's also part of the file group of classes